versioned_docs/version-8.5/apis-tools/spring-zeebe-sdk/getting-started.md

```diff
@@ -103,6 +103,7 @@ Example of configuring the connection to a Self-Managed Zeebe cluster:
 ```properties
 zeebe.client.cloud.clientId=your-client-id
 zeebe.client.cloud.clientSecret=your-client-secret
+zeebe.client.cloud.scope=your-scope # optional
```

Comment:
This is not released in 8.5 see the original issue https://github.com/camunda/camunda/issues/23071 . It will be included in the next 8.5 patch.

We need to revert this.

Please always refer to the actual originating issue for context.

Thanks @akeller for the ping.